packages/environment/test/develop.js

```diff
@@ -29,14 +29,10 @@ describe("Environment develop", function () {
         gas: expectedNetwork.gas
       }
     };
-    sinon.stub(Environment, "detect");
-  });
-
-  afterEach("Restore Environment detect", async function () {
```

Comment:
Is there a reason for deleting this?  Do we not want to restore after each test?

Reply:
Oh I guess we don't want to restore if it's not a stub in the first place...

Reply:
I undid the commit that did this factor because it was unrelated to the race.